python/nav/models/manage.py

```diff
@@ -1576,10 +1600,18 @@ class Arp(models.Model):
 
     id = models.AutoField(db_column='arpid', primary_key=True)
     netbox = models.ForeignKey(
-        'Netbox', on_delete=models.CASCADE, db_column='netboxid', null=True
+        'Netbox',
+        on_delete=models.CASCADE,
+        db_column='netboxid',
+        null=True,
+        related_name="arps",
```

Comment:
Please, no. ARP is an acronym, `arps` just sounds like someone is burping. `arp_set` is good enough.